scripts/scripts.js

```diff
@@ -28,6 +28,7 @@ const CONFIG = {
     '': { ietf: 'en-US', tk: 'hah7vzn.css' },
     de: { ietf: 'de-DE', tk: 'hah7vzn.css' },
     kr: { ietf: 'ko-KR', tk: 'zfo3ouc' },
+    jp: { ietf: 'jp-JP', tk: 'dvg6awq' },
```

Comment:
I believe the language code is ja
```suggestion
    jp: { ietf: 'ja-JP', tk: 'dvg6awq' },
```